cluster/ci/config/mirroring/origin_v4_0_quay

```diff
@@ -1,6 +1,7 @@
 registry.svc.ci.openshift.org/openshift/origin-v4.0:ansible quay.io/openshift/origin-ansible:v4.0 quay.io/openshift/origin-ansible:v4.0.0 quay.io/openshift/origin-ansible:latest
 registry.svc.ci.openshift.org/openshift/origin-v4.0:artifacts quay.io/openshift/origin-artifacts:v4.0 quay.io/openshift/origin-artifacts:v4.0.0 quay.io/openshift/origin-artifacts:latest
 registry.svc.ci.openshift.org/openshift/origin-v4.0:aws-machine-controllers quay.io/openshift/origin-aws-machine-controllers:v4.0 quay.io/openshift/origin-aws-machine-controllers:v4.0.0 quay.io/openshift/origin-aws-machine-controllers:latest
+registry.svc.ci.openshift.org/openshift/origin-v4.0:baremetal-machine-controllers docker.io/openshift/origin-baremetal-machine-controllers:v4.0 docker.io/openshift/origin-baremetal-machine-controllers:v4.0.0 docker.io/openshift/origin-baremetal-machine-controllers:latest
```

Comment:
Looks like docker needs to change to quay in this one

Reply:
Good catch, fixed